backend/curriculum_tracking/models.py

```diff
@@ -1667,6 +1667,35 @@ def request_user_is_assignee(self):
 
         return self.assignees.first() == user
 
+    def request_user_can_request_review(self, user=None):
+        """
+        Check if current user can request review for this card
+        """
+        if self.status not in [AgileCard.IN_PROGRESS, AgileCard.REVIEW_FEEDBACK]:
```

Comment:
would it not make sense to return false if this is not a project card? Because nobody can request a review on a topic?

like: 
```
if card.content_type_nice !== project:
    return False
```

This would simplify code elsewhere and make this function more explicit and cohesive.